dataeng/resources/monitor-bigquery-loading.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+env
+
+VENV_ROOT=$WORKSPACE/venvs
+mkdir -p $VENV_ROOT
+
+virtualenv $VENV_ROOT/analytics-tools
+TOOLS_BIN=$VENV_ROOT/analytics-tools/bin
+. $TOOLS_BIN/activate
+
+pip install -r ${WORKSPACE}/analytics-tools/bigquery-monitoring/requirements.txt
+
+# download credentials file to temporary directory
+aws s3 cp $CREDENTIALS /tmp/bigquery_credentials.json
+
+export PARTITION_DATE=$(date +%Y-%m-%d -d "$PARTITION_DATE")
+
+. ${WORKSPACE}/analytics-tools/bigquery-monitoring/bigquery_monitor_event_loading.sh
```

Comment:
This script doesn't yet exist in a PR, correct?

Reply:
should be here: https://github.com/edx/analytics-tools/pull/13/files#diff-c5790760e0c72ae4fd6039c8137f2e00

Reply:
That's also where `$DATASET` is used.